doc/common/glossary.md

```diff
@@ -20,94 +20,136 @@ To help guide the reader, this glossary provides an introduction to the terminol
-- **External Network**: External networks provide network connectivity for a cloud infrastructure tenant to resources outside of the tenant space.
-- **Quota**: An imposed upper limit on specific types of resources, usually used to prevent excessive resource consumption in the **VIM** by a given consumer (tenant).
-- **Resource pool**: A logical grouping of cloud infrastructure hardware and software resources. A resource pool can be based on a certain resource type (for example, compute, storage, network) or a combination of resource types. An **Cloud Infrastructure** resource can be part of none, one or more resource pools.
-- **Compute Node**: An abstract definition of a server.
+
+- **Compute Node:** An abstract definition of a server.
 >_*Note:*_ A compute node can refer to a set of hardware and software that support the VMs or Containers running on it.
-- **Service Assurance (SA)**: collects alarm and monitoring data. Applications within SA or interfacing with SA can then use this data for fault correlation, root cause analysis, service impact analysis, SLA management, security, monitoring and analytic, etc.
+
+- **External Network:** External networks provide network connectivity for a cloud infrastructure tenant to resources outside of the tenant space.
+
+- **Multi-tenancy:** feature where physical, virtual or service resources are allocated in such a way that multiple tenants and their computations and data are isolated from and inaccessible by each another. ([ETSI](https://www.etsi.org/deliver/etsi_gs/NFV/001_099/003/01.04.01_60/gs_nfv003v010401p.pdf))
+
+- **Quota:** An imposed upper limit on specific types of resources, usually used to prevent excessive resource consumption in the **VIM** by a given consumer (tenant).
+
+- **Resource pool:** A logical grouping of cloud infrastructure hardware and software resources. A resource pool can be based on a certain resource type (for example, compute, storage, network) or a combination of resource types. An **Cloud Infrastructure** resource can be part of none, one or more resource pools.
+
+- **Service Assurance (SA):** collects alarm and monitoring data. Applications within SA or interfacing with SA can then use this data for fault correlation, root cause analysis, service impact analysis, SLA management, security, monitoring and analytic, etc.
+
+- **Tenant:** One or more cloud service users sharing access to a set of physical and virtual resources ([ITU](https://www.itu.int/rec/dologin_pub.asp?lang=e&id=T-REC-Y.3500-201408-I!!PDF-E&type=items)).
+>_*Note*_ Tenants represent an independently manageable logical pool of compute, storage and network resources abstracted from physical hardware.
+
+- **Tenant (Internal) Networks:** Virtual networks that are internal to tenant instances.
```

Comment:
there is no definition for "tenant instance". Instance has been previously defined as:

Instance: is a virtual compute resource, in a known state such as running or suspended, that can be used like a physical server.

Given the above definition, I don't think that is what is meant by "tenant instances".

Reply:
Thanks for the suggestion! Let`s fix this in another PR.

Reply:
We have created #1907 to fix this.

Reply:
@pgoyal01, can we resolve this thread?